packages/storybook8/stories/Concepts/TeamsInterop/Lobby/Lobby.stories.tsx

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { Meta } from '@storybook/react';
+import React from 'react';
+
+import { controlsToAdd } from '../utils/controlsUtils';
+import { Lobby as LobbyComponent } from './snippets/Lobby.snippet';
+
+const LobbyStory: (args) => JSX.Element = (args) => {
+  return <LobbyComponent {...args} />;
+};
+
+export const Lobby = LobbyStory.bind({});
+
+export default {
+  title: 'Concepts/Teams Interop/Lobby',
+  component: Lobby,
+  argTypes: {
+    callStateText: controlsToAdd.callStateText,
+    callStateSubText: controlsToAdd.callStateSubText,
+    isVideoReady: controlsToAdd.isVideoReady
+  }
+} as Meta;
```

Comment:
Preview not rendering properly:
<img width="1090" alt="Screenshot 2024-07-29 at 3 44 54 PM" src="https://github.com/user-attachments/assets/597a9449-6a11-4fe8-892c-086c93ec750b">
this is how it looks in storybook6:
<img width="1475" alt="Screenshot 2024-07-29 at 3 44 45 PM" src="https://github.com/user-attachments/assets/0a7db32c-b2d2-4779-b078-522c814eb587">

Reply:
will look into this

Reply:
turns out we need to add our "Stack" with the styles we are using our composite container stylings! 

added it in now